test/cover/test_netcvode.py

```diff
@@ -562,7 +564,9 @@ def run1(key):
     def run2(key):
         h.finitialize(-65)
         cv.solve(2)
-        chk(key + " tvec size", tvec.size())
+        # Note the very large tolerance, this seems relatively unstable with the
+        # NVIDIA compilers
+        chk(key + " tvec size", tvec.size(), tol=0.25)
```

Comment:
@nrnhines in my local test I saw
```
32: jacobian 2 tvec size difference
32: std =  93
32: val =  118
```
which naïvely seems quite large. If I understand correctly, this is saying that the NVHPC build uses 25% more time steps than the reference, which I guess was built with GCC. Does this seem reasonable to you?

Reply:
Just a guess... the problem may be insufficient initialization of Cvode with respect to dt which may be left over from the previous test. (I remember burning a couple hours on one of those already in this context). Need to see what happens when we always start out with ```h.dt = 0.025````.

Reply:
Created https://github.com/neuronsimulator/nrn/issues/1823 to follow up.